### DISCUSSION TITLE
Chocolatey Cake.Portable Package 1.3.0 is unlisted.

### DISCUSSION THREAD

**Discussion by xforever1313** [cake-build/cake#3874] [Q&A] [answered]
My group is still using Cake v1.X and still use the Chocolatey package, Cake.Portable.

I noticed when installing cake on a new PC today that Chocolatey was only installing Cake 1.2.0, even though there is a 1.3.0 version of the package.

When I went to the Chocolatey page, I noticed that the 1.3.0 version of the package was unlisted: https://community.chocolatey.org/packages/cake.portable/1.3.0

Was that done on purpose?  If so why?  While I can install it via "choco install Cake.Portable -Version 1.3.0", I'm mainly wondering if there was something horribly wrong with 1.3.0 where we should not install it?

Thanks!

**Comment by gep13** [comment, marked as answer]
@xforever1313 Thanks for bringing this up!

We have chatted in the team, and we are not aware of any reason as to _why_ this was unlisted, so I have just went in and listed this package version.

Let us know if you run into any problems.

Our recommendation going forward is that folks switch to using the .NET Global Tool version of Cake.  Is this something that you are thinking about doing?

**Reply by xforever1313** [reply]
Thanks!  It looks like its listed now 🎉.

At some point, yeah.  We need to finish migrating off of .NET Framework to .NET 6 so everyone has dotnet.exe installed on their computer first :P.
